docs/site/Memory-connector.md

```diff
@@ -34,24 +31,6 @@ own isolated data not shared in the cluster.
 You can persist data between application restarts using the `file` property. See
 [Data persistence](#data-persistence) for more information. " %}
 
-## Creating a data source
-
-By default, an application created with the
-[Application generator](Application-generator.html) has a memory data source
-defined; for example:
-
-{% include code-caption.html content="/server/datasources.json" %}
-
-```javascript
-"db": {
-    "name": "db",
-    "connector": "memory"
-}
-```
-
-Use the [Data source generator](Data-source-generator.html) to add a new memory
```

Comment:
do we want to keep the "creating a data source" section here... with the info to use the data source generator to create a datasource making use of this connector?

Reply:
I feel that "Creating a DataSource" may be redundant to the [DataSource](https://github.com/strongloop/loopback-next/blob/1fd148f1f0e8630c2159edee62a1d489480f4e9d/docs/site/DataSource.md) docs.